src/initPressNews.js

```diff
@@ -0,0 +1,36 @@
+import { ALL_ECONOMY, FIRST_NEWS_PAGE } from "./constant.js";
+import { fetchpressNews } from "./dataFetch.js"
+import turnNewsPage from "./turnNewsPage.js";
+import showNews from "./showNews.js";
+import showNewsOfCategory from "./clickCategory.js";
+import showProgress from "./showProgress.js";
+
+const pressNewsList = [[], [], [], [], [], [], []];
+const shuffledPressNews = [[], [], [], [], [], [], []];
+
+/**
+카테고리 별 언론사 순서 랜덤으로 섞기
+ */
+async function randomizeNews() {
+  const pressNewsData = await fetchpressNews();
+  const category = ["종합/경제", "방송/통신", "IT", "영자지", "스포츠/연예", "매거진/전문직", "지역"];
+
+  pressNewsList.forEach((arr, idx) => {
+    pressNewsList[idx] = pressNewsData.filter(press => press["category"] === category[idx]);
+    shuffledPressNews[idx] = [...pressNewsList[idx]].sort(() => Math.random() - 0.5);
+  })
+  return category;
+}
```

Comment:
```suggestion
async function randomizeNews() {
  const pressNewsData = await fetchpressNews();
  const categories = ["종합/경제", "방송/통신", "IT", "영자지", "스포츠/연예", "매거진/전문직", "지역"];

  const pressNewsList = categories.map(category => pressNewsData.filter(press => press["category"] === category));
  const shuffledPressNews = pressNewsList.map(list => [...list].sort(() => Math.random() - 0.5));

  return categories;
}
```
저 두 변수를 다른데서사용하는게 아니라면 함수 내에 선언해서 사용해도 될 것 같아요 forEach사용하는 함수도 조금 구분을 하는게 좋아보여요